molsketch/net.sourceforge.molsketch.appdata.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2017 Hendrik Vennekate <mail_AT_domain.tld> -->
```

Comment:
Alexander, don't you want to own the copyright yourself? (Otherwise, please remove the email address placeholder -- or am I required to fill that in?)

Reply:
Just add your own. You can also use the "_AT_" trick.

Reply:
Ok, will do.